addons/flannel/0.21.5/install.sh

```diff
@@ -269,15 +282,18 @@ function weave_to_flannel() {
     # Get the list of namespaces
     namespaces=$(kubectl get ns -o name | grep -Ev '(kube-system|longhorn-system|rook-ceph|openebs|kube-flannel)' | cut -f2 -d'/')
 
-    for ns in ${namespaces}; do
-        # Delete pods with a grace period and timeout, and ignore errors
-        timeout 80 kubectl delete pods -n "${ns}" --all --grace-period=60 || logWarn "Timeout reached while trying to delete pods in namespace: ${ns}"
+    for ns in "${namespaces[@]}"; do
+        log "Waiting up to 5 minutes to delete all pods from ${ns}"
+        if ! spinner_until 300 kubectl -n ${ns} delete pods --all --ignore-not-found; then
+            logWarn "Timeout reached while trying to delete pods in namespace: ${ns}"
+        fi
+    done
 
-        # Use the spinner_until function to wait until all pods are deleted
-        if ! spinner_until 60 flannel_pods_remain "${ns}"; then
+    for ns in "${namespaces[@]}"; do
+        log "Waiting up to 2 minutes to verify if pods will be terminated"
+        if ! spinner_until 120 flannel_pods_remain ${ns}; then
             # If pods remain after the timeout, force delete them
-            logWarn "Force deleting remaining pods in namespace: ${ns}"
-            kubectl -n "${ns}" delete pods --all --grace-period=0 --force || true
+            logWarn "The namespace ${ns} has pods which were not terminated"
```

Comment:
if you're going to change the logic, you can't just leave the comment describing the logic the same.

I also disagree with removing the force deletion here.

Reply:
@laverya the force was added in the PR https://github.com/replicatedhq/kURL/pull/4568/files
However, besides it sort out some scenarios I found others ones where the cluster will end up broken
We cannot use force .